lib/pgBackRest/Backup/Backup.pm

```diff
@@ -95,7 +95,7 @@ sub fileNotInManifest
     foreach my $strName (sort(keys(%{$hFile})))
     {
         # Ignore certain files that will never be in the manifest
-        if ($strName eq FILE_MANIFEST ||
+        if ($strName eq FILE_MANIFEST_COPY ||
             $strName eq '.')
```

Comment:
backup.manifest should never be present for a resumed backup.  If it is it should be removed.